plenum/test/node_catchup_with_3pc/test_stashing_3pc_while_catchup_checkpoints.py

```diff
@@ -22,7 +22,14 @@
 
 CHK_FREQ = 5
 
-@pytest.mark.skip("in progress")
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    old_btch_sz = tconf.Max3PCBatchSize
+    tconf.Max3PCBatchSize = 1
+    yield tconf
+    tconf.Max3PCBatchSize = old_btch_sz
```

Comment:
There is `max_3pc_batch_limits` context manager in test helpers that can be used here:
```python
def tconf(tconf):
    with max_3pc_batch_limits(tconf, size=1) as tconf:
        yield tconf
```

Reply:
Done